network/p2p/gossip/gossipable.go

```diff
@@ -7,9 +7,13 @@ import "github.com/ava-labs/avalanchego/ids"
 
 // Gossipable is an item that can be gossiped across the network
 type Gossipable interface {
-	GetID() ids.ID
-	Marshal() ([]byte, error)
-	Unmarshal(bytes []byte) error
+	GetGossipID() ids.ID
```

Comment:
I don't think golang generally uses getters (the `Get` prefix).
```suggestion
	GossipID() ids.ID
```